test/tools/OpenCover/OpenCover.psm1

```diff
@@ -506,13 +506,47 @@ function Compare-CodeCoverage
     }
 
     $change = Get-CodeCoverageChange -r1 $Run1 -r2 $Run2 -Class $ClassName
-    if ( $Summary -or $ClassName ) 
-    { 
-        $change 
+    if ( $Summary -or $ClassName )
+    {
+        $change
     }
     else
-    { 
-        $change.Deltas 
+    {
+        $change.Deltas
+    }
+}
+
+function Compare-FileCoverage
+{
+    param (
+        [Parameter(Position=0,Mandatory=$true)]$ReferenceCoverage,
+        [Parameter(Position=1,Mandatory=$true)]$DifferenceCoverage,
+        [Parameter(Position=2,Mandatory=$true)]$FileName
+    )
+    # create a couple of hashtables where the key is the path
+    # so we can compare file coverage
+    $reference = $ReferenceCoverage.GetFileCoverage($FileName) | Foreach-Object { $h = @{} } { $h[$_.path] = $_ } {$h}
+    $difference = $differenceCoverage.GetFileCoverage($FileName)| Foreach-Object { $h = @{}}{ $h[$_.path] = $_ }{$h }
```

Comment:
should have a space before the pipe

Reply:
fixed

---
In reply to: [137402201](https://github.com/PowerShell/PowerShell/pull/4764#discussion_r137402201) [](ancestors = 137402201)